.github/workflows/pr.yaml

```
@@ -0,0 +1,316 @@
+name: Galaxy Tool Linting and Tests for push and PR
+on: [push, pull_request]
+env:
+  GALAXY_REPO: https://github.com/galaxyproject/galaxy
+  GALAXY_RELEASE: release_20.01
+jobs:
+  setup:
+    name: Setup cache and determine changed repositories
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Cache .cache/pip
+      uses: actions/cache@v1
+      id: cache-pip
+      with:
+        path: ~/.cache/pip
+        key: pip_cache_py${{ matrix.python-version }}_${{ env.GALAXY_RELEASE }}
+    - name: determine latest galaxy commit
+      run: echo ::set-env name=GALAXY_HEAD_SHA::$(git ls-remote ${{ env.GALAXY_REPO }} refs/heads/${{ env.GALAXY_RELEASE }} | cut -f1)
+    - name: Cache .planemo
+      uses: actions/cache@v1
+      id: cache-planemo
+      with:
+        path: ~/.planemo
+        key: planemo_cache_py${{ matrix.python-version }}_${{ env.GALAXY_HEAD_SHA }}
+    - name: install Planemo and flake8
+      run: pip install planemo flake8
+    - name: fake a planemo run to update cache
+      if: steps.cache-pip.outputs.cache-hit != 'true' && steps.cache-planemo.outputs.cache-hit != 'true'
```

Shouldn't this be an OR?

```suggestion
      if: steps.cache-pip.outputs.cache-hit != 'true' || steps.cache-planemo.outputs.cache-hit != 'true'
```